Soccar [Physics]/src/nl/soccar/physics/GameEngine.java

```diff
@@ -72,6 +74,22 @@ public void stop() {
         timer.cancel();
     }
 
+    public void addListener(GameEventListener listener) {
+        Objects.requireNonNull(listener);
+
+        synchronized (listeners) {
+            listeners.add(listener);
+        }
+    }
+
+    public void removeListener(GameEventListener listener) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)